packages/block-template/webpack-dev-server.config.js

```diff
@@ -31,6 +31,7 @@ module.exports = {
         "sentry-trace",
       ],
     },
+    open: process.env.BROWSER !== "none",
```

Comment:
Replaces `--open` in webpack CLI. `BROWSER=none` matches CRA behavior: https://create-react-app.dev/docs/advanced-configuration/ (I saw it somewhere else too).

I use this flag inside `smoke-test-create-block-app` script to avoid annoying window jumps.